api/src/main/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceImpl.java

```diff
@@ -205,14 +217,55 @@ public WorkspaceAdminView getWorkspaceAdminView(String workspaceNamespace) {
         .resources(adminWorkspaceResources);
   }
 
+  @Override
+  public List<ListRuntimeResponse> deleteRuntimesInWorkspace(
+      String workspaceNamespace, ListRuntimeDeleteRequest req) {
+    final String googleProject =
+        getWorkspaceByNamespaceOrThrow(workspaceNamespace).getGoogleProject();
```

Comment:
I'm assuming this is the only change to the implementation here

Reply:
yup. Extra db lookup to get googleProject